loco-cli/src/git.rs

```diff
@@ -144,6 +144,23 @@ fn git_exists() -> bool {
     }
 }
 
+pub fn is_a_git_repo(destination_path: &PathBuf) -> eyre::Result<bool> {
+    let destination_path = destination_path.canonicalize()?;
+    match Command::new("git")
+        .arg("-C")
+        .arg(destination_path)
+        .arg("rev-parse")
+        .arg("--is-inside-work-tree")
+        .output()
+    {
+        Ok(output) => Ok(output.status.success()),
```

Comment:
let logged the error if the command returned an error
```suggestion
     Ok(output) => {
            if output.status.success() {
                Ok(true)
            } else {
                tracing::error!(
                    error = tracing::field::debug(output),
                    "git command returned an error"
                );
                Ok(false)
            }
        }
```

Reply:
Id the command not succeeded let print the strerr to log.